docs/core/config/auth/databases/passwd_file.md

```diff
@@ -64,6 +64,11 @@ passdb passwd-file {
 }
 ```
 
+[[added,passwd_file_iteration]]
+
+To iterate databases that use variable expansion in the path, you need to provide enough information when using
+[[doveadm,user]] to iterate. Example [[doveadm,user,*@domain.com]] would match with ``%{user | domain}`` expansion.
```

Comment:
This example could also be more explicit, showing an example passdb passwd_file configuration pointing to domains.

Reply:
Oh, there is an example above. Then the text could say maybe `would match the %{user | domain} expansion in the above example passdb passwd-file.`